content/W2015/W15-blog-20150105.md

```diff
@@ -12,6 +12,6 @@ on GitHub](https://github.com/wics-uw/website). You can find documentation on
 how to contribute the site in the README.
 
 We'll be occasionally posting news and other information on our 
-[blog](/category/blog.html). Check out the [Events 
-section](/category/events.html) for our upcoming events, and the [Media 
-section](/category/media.html) for recordings of past talks.
+[blog](/category/blog/). Check out the [Events section](/category/events/) for
+our upcoming events, and the [Media section](/category/media/) for recordings
```

Comment:
Had to fix these links. Would have loved to use the `{category}category_name` style linking but that's actually what I submit an upstream patch to Pelican for (functionality currently broken).